docs/deployment/landing_page.rst

```diff
@@ -15,6 +15,38 @@ your organization.
           implement minimum security requirements is sure to be noticed, and
           could undermine trust, discouraging possible sources.
 
+SecureDrop `maintains a directory of instances that meet our strict guidelines.
+<https://securedrop.org/directory/>`__ If you would like to be considered for
+inclusion in this directory, make sure your landing page is in compliance with
+the requirements below, then `send us a request using this form.
+<https://securedrop.org/directory/submit/>`__
+
+There are several benefits to being included in the SecureDrop directory. The
+most significant benefit is that it will be easier for potential sources to
+find your SecureDrop instance. Additionally, being included in the directory
+makes you eligible for a short onion address. This improves the experience by
+turning a lengthy, non-descriptive address into one that is short and 
+memorable. For example, a long .onion address might look like: ::
+
+    xp44cagis447k3lpb4wwhcqukix6cgqokbuys24vmxmbzmaq2gjvc2yd.onion
+
+whereas the short onion address might look like: ::
+
+     theguardian.securedrop.tor.onion
```

Comment:
Let's pick a fictional example here (both to avoid accidentally having an old SD onion address in the docs at some future point, and as a rule of thumb to only reference external organizations with their explicit knowledge). We can use the same example as we did here: https://securedrop.org/faq/getting-onion-name-your-securedrop/#what-are-onion-names